packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -10,18 +10,22 @@ const { expect } = chai;
 describe('Collection aggregations tab', function () {
   let compass: Compass;
   let browser: CompassBrowser;
+  let screenshot: string | undefined;
 
   before(async function () {
+    screenshot = 'collection-aggregations-tab-before';
     compass = await beforeTests();
     browser = compass.browser;
 
     await browser.connectWithConnectionString('mongodb://localhost:27018/test');
 
     await browser.navigateToCollectionTab('test', 'numbers', 'Aggregations');
+
+    screenshot = undefined;
```

Comment:
I don’t necessarily mind this bookkeeping, but I’m wondering if there’s a better way. For one, I wouldn’t necessarily consider it bad if we just always captured a screenshot on failure, and maybe we could base the screenshot name on the test name? That information should be accessible, e.g.:

https://github.com/mongodb-js/mongosh/blob/5afb80349eab42e026c5c8219cf378c7933b0d66/packages/cli-repl/test/e2e.spec.ts#L19

https://github.com/mongodb-js/mongosh/blob/5afb80349eab42e026c5c8219cf378c7933b0d66/packages/cli-repl/test/test-shell.ts#L79

Reply:
you can do this.currentTest.failed (which we do in the afterTest handler for afterEach), but this.currentTest.title is not very helpful. it will be `the describe test / the next test that would have run` at best. Which I suppose I could try and use. ie. cut off after the `/` and run some regex on it all to try and turn into a filename..